settings.gradle

```diff
@@ -1,13 +1,13 @@
-rootProject.name = 'paparazzi-parent'
+rootProject.name = 'paparazzi-root'
 
 include ':sample'
-include ':paparazzi'
-include ':paparazzi-agent'
-include ':paparazzi-gradle-plugin'
-include ':libs:layoutlib'
-include ':libs:native-macarm'
-include ':libs:native-macosx'
-include ':libs:native-win'
-include ':libs:native-linux'
 
-enableFeaturePreview('TYPESAFE_PROJECT_ACCESSORS')
\ No newline at end of file
+includeBuild('paparazzi') {
+  dependencySubstitution {
+    substitute module('app.cash.paparazzi:paparazzi') with project(':paparazzi')
```

Comment:
`with` is deprecated now. Any reason we can't use `using` instead?

Reply:
yep, you're right.  will update.